src/guacd/conf-file.h

```diff
@@ -24,6 +24,11 @@
 
 #include "conf.h"
 
+/**
+ * Creates a new configuration structure, filled with default settings.
+ */
+guacd_config* guacd_conf_create();
+
```

Comment:
After changes, guacd_config creation is independent of parsing configuration files.
Should I put this function here or in `conf.h` (and implementation in **new** `conf.c` file)?

Reply:
Even though it's independent of the parsing, it still seems like it's related to pulling values from the files, so I'm fine with it staying where it is.